cluster/manifests/coredns-local/service.yaml

```diff
@@ -0,0 +1,21 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: coredns-local-dnsmasq
+  namespace: kube-system
+  labels:
+    application: coredns
```

Comment:
This overlaps with the current CoreDNS as deployment. Does it interfere on deployment? Do we have hairpin mode set?

Reply:
I don't think the service labels matter, and the selector includes `instance: node-dns`. In my tests it didn't interfere with anything.

Reply:
The service is only there for the metrics collection anyway.

Reply:
ok, if it's only metrics collection we can go